herokuappTests/auth/authenticator.py

```diff
@@ -0,0 +1,10 @@
+import requests
+
+class Authenticator:
+    def get_token(self):
+        payload = {
+            "username": "admin",
+            "password": "password123"
+        }
+        response = requests.post("https://restful-booker.herokuapp.com/auth", json=payload)
+        return response.json().get("token")
```

Comment:
You should create some handle here if token is not retrieved successfully. For example using status code to decide what to retrieve since this return will fail in json parsing if request is not successful